src/lib/stores/time-format.ts

```diff
@@ -6,6 +6,20 @@ import { getLocalTimezone } from '$lib/utilities/format-date';
 
 type TimeFormatTypes = 'relative' | 'absolute';
 
+export const timestampFormats = {
+  short: 'do MMM yyyy H:mm:ss.SS',
+  medium: 'yyyy-MM-dd z HH:mm:ss.SS',
+  long: 'MMMM do yyyy, hh:mm:ss.SS a z',
+  abbreviated: 'yyyy-MM-dd HH:mm:ss a',
+  abbreviatedWithoutSeconds: 'yyyy-MM-dd HH:mm a',
+} as const;
```

Comment:
Wondering if we should use 'full' | 'long' | 'medium' | 'short' and map to Intl.DateTimeFormat. What I don't know is if our timezone's locale map correctly.

```
new Intl.DateTimeFormat('en-US', {
  year: 'numeric',
  month: 'long',
  day: 'numeric'
}).format(new Date()); // "November 17, 2025"

// Short date
new Intl.DateTimeFormat('en-US', {
  year: '2-digit',
  month: '2-digit',
  day: '2-digit'
}).format(new Date()); // "11/17/25"

// Date + Time
new Intl.DateTimeFormat('en-US', {
  dateStyle: 'short',
  timeStyle: 'short'
}).format(new Date()); // "11/17/25, 2:30 PM"
Key options:

dateStyle: 'full' | 'long' | 'medium' | 'short'
timeStyle: 'full' | 'long' | 'medium' | 'short'
```

Reply:
What is the default? Medium? Just might be weird for us 12-hour folks to have that now be 24